src/hooks/useClaimOrderCallback.ts

```diff
@@ -1,19 +1,21 @@
-import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useCallback, useEffect, useRef, useState } from 'react'
 import { TokenAmount } from 'uniswap-xdai-sdk'
 
 import { BigNumber } from '@ethersproject/bignumber'
 import { Contract } from '@ethersproject/contracts'
 
-import { useSingleCallResult } from '../state/multicall/hooks'
 import { DerivedAuctionInfo } from '../state/orderPlacement/hooks'
 import { AuctionIdentifier } from '../state/orderPlacement/reducer'
 import { useHasPendingClaim, useTransactionAdder } from '../state/transactions/hooks'
 import { ChainId, calculateGasMargin, getEasyAuctionContract } from '../utils'
+import { getLogger } from '../utils/logger'
 import { additionalServiceApi } from './../api'
 import { decodeOrder } from './Order'
 import { useActiveWeb3React } from './index'
 import { useGasPrice } from './useGasPrice'
 
+const logger = getLogger('useClaimOrderCallback')
```

Comment:
Is it still needed?

Reply:
I am not sure if we have an error handling policy but I thought it might be useful to register the [unavailability of EasyAuction services](https://github.com/gnosis/ido-ux/pull/592/files#diff-99c45e8374392fb1f48169b1d9b6ba52c3e15e2666b8b4e1ab940ab5ca1a2ac1R264).

if you find it unnecessary, please let me know.